src/pages/home/models-list/models-list.style.js

```diff
@@ -11,7 +11,7 @@ export const useStyles = makeStyles((theme) => ({
     justifyContent: 'center',
     transition: 'height 1s ease',
     '@media (max-width: 568px)': {
-      maxHeight: '600px'
+      maxHeight: '1000px'
```

Comment:
Remove all height values for this element.

Reply:
fixed